### ISSUE TITLE
1.18 Questions

### ISSUE THREAD

**Issue by AMGitsKriss** [Zodsmar/SeedSearcherStandaloneTool#129]
Now that the 1.18 seeds have supposedly been finalised, I'm going to ask the dreaded question:

What will need doing for 1.18? And anything suitable for a newer contributor?

**Comment by Capitalplayground** [comment]
He will probably only wait till the official release.

**Comment by Suklaam0nni** [comment]
I understand the seeds for 1.18 should stay fixed now. If it is possible to update the tool before the official release that would be awesome! As a server host, my community is eagerly waiting to start a new world when 1.18 officially drops. If I could scout out seeds before the official release, that would be optimal.

Also I wanted to ask if it would be a big chance to add range to the search width, so I could search for things for example in between 1000 and 2000 blocks from spawn.

Optimally I would also like to search for different set of biomes and structures for multiple ranges like that, but it sounds like a bigger change.

Anyway thank you so much for this awesome tool!

**Comment by Zodsmar** [comment]
I do plan on updating to 1.18. I also have my own server (for friends and family) which I also will need a 1.18 seed for. I am looking into getting it updated ASAP. I can't promise that it will be out before the official release but I would like to be either before the release or just after

**Comment by Capitalplayground** [comment]
Just wana say thanks Zodsmar, your little seed finder is just the best out there for finding what you want.

**Comment by qumielo** [comment]
> I do plan on updating to 1.18. I also have my own server (for friends and family) which I also will need a 1.18 seed for. I am looking into getting it updated ASAP. I can't promise that it will be out before the official release but I would like to be either before the release or just after

Great to hear you're working on it!
Can't wait to use it for searching 1.18 seeds

**Comment by Zodsmar** [comment]
@AMGitsKriss @qumielo @Capitalplayground @Suklaam0nni I just wanted to give you guys an update. The API's still aren't updated to 1.18 because in the last few release candidates they messed with ore generation etc. which broke the API's. Once they get updated I can get the tool out. So it is still coming just delayed (out of my control).

**Comment by Egamer25** [comment]
Zodsmar,  Thanks for the update.   I know it can be frustrating when Mojang comes out with a new version and they break the API's hope they get them fixed soon.  Absolutely love the tool and look forward to when Mojang gets the API's fixed!

**Comment by Zodsmar** [comment]
> Zodsmar, Thanks for the update. I know it can be frustrating when Mojang comes out with a new version and they break the API's hope they get them fixed soon. Absolutely love the tool and look forward to when Mojang gets the API's fixed!

Thanks! I wanted to make a correction. The API's are not official Mojang API's they are custom-made. They were done and ready to go until Mojang made the change. But yes I am also looking forward to the apis being updated! :)

**Comment by Amelia-Trundle** [comment]
hey i'd just like to ask if there are any updates on the progress towards a 1.18 release, seeing as its been a little inactive. i don't want to seem like I'm pressuring i'm mostly just curious to what is going on behind the scenes :)

**Comment by Zodsmar** [comment]
@Amelia-Trundle Hey. Yeah so the tool itself is pretty much ready for 1.18 Im still just waiting on the API updates. So its out of my hands... whenever those get updated I can finish and release the tool

**Comment by Amelia-Trundle** [comment]
awesome thanks for the update on progress :)

**Comment by rithik25** [comment]
Hello,

Are the API updates not out yet?

**Comment by Zodsmar** [comment]
> Hello,
> 
> Are the API updates not out yet?

nope

**Comment by rithik25** [comment]
Just curious, do you know where these other seed applications like Cubiomes or Chunkbase get their data from for 1.18? Do they not use the API as well?

**Comment by Zodsmar** [comment]
They use different api's to my knowledge.

**Comment by rithik25** [comment]
> They use different api's to my knowledge.

May I know which API you use for SASSA?

Thank you for the great work btw

**Comment by Zodsmar** [comment]
@rithik25 the apis are in the build.gradle but essentially these https://github.com/SeedFinding (MCBiome is the main one) and if you see it hasnt been updated since november. All I know is they should be updated soon *TM*

**Comment by rithik25** [comment]
Alright thank you.

**Comment by LPZO** [comment]
any mew information on 1.18 update?

**Comment by LPZO** [comment]
Can't wait for the update. this is the best seed finder out there
Good luck

**Comment by LudusLight** [comment]
Hate to keep poking this, but has there been any change?

**Comment by lajawi** [comment]
> Hate to keep poking this, but has there been any change?

Doesn't seem like it, the files from the API are still not updated, still says November 2021

**Comment by Zodsmar** [comment]
> > Hate to keep poking this, but has there been any change?
> 
> Doesn't seem like it, the files from the API are still not updated, still says November 2021

Yep at at this point I doubt it will be updated. I wouldn't mind working on SASSA again but I don't have the time to create the API's from scratch nor do I really want too. So until a point that either new API's become available this project is essentially complete for now.
